libcxx/test/std/utilities/expected/expected.void/observers/value.lwg3940.compile.fail.cpp

```diff
@@ -0,0 +1,25 @@
+//===----------------------------------------------------------------------===//
```

Comment:
We'd like to avoid writing compile.fail.cpp. Instead, verify.cpp is better because it can check the exact line that causes the compilation error. an example can be found here:

https://github.com/llvm/llvm-project/blob/main/libcxx/test/libcxx/utilities/expected/expected.unexpected/swap.mandates.verify.cpp

Reply:
Great idea! Thank you.

Done. Could you review it again when you have any free time?